pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = hydroshare/settings.py
-python_files = test/functional/test*
+python_files = pytest/functional*
 filterwarnings =
```

Comment:
make it obvious from the top-level directory that these are pytest and not nose or selenium